app/controllers/idv/personal_key_controller.rb

```diff
@@ -10,20 +10,25 @@ class PersonalKeyController < ApplicationController
     before_action :confirm_profile_has_been_created
 
     def show
-      analytics.idv_personal_key_visited
+      analytics.idv_personal_key_visited(proofing_method: proofing_method)
       add_proofing_component
 
       finish_idv_session
     end
 
     def update
       user_session[:need_personal_key_confirmation] = false
-      analytics.idv_personal_key_submitted
+
+      analytics.idv_personal_key_submitted(proofing_method: proofing_method)
       redirect_to next_step
     end
 
     private
 
+    def proofing_method
+      "#{user_session['idv']['address_verification_mechanism']} verification"
```

Comment:
1. I would use `.dig` or `.&` the null-safe navigator to make sure this doesn't blow up
2. if this is only used for logging, would it make sense to just use the "raw" value and not interpolate `" verification"` after it?

```suggestion
      user_session.dig('idv', 'address_verification_mechanism')
```

Reply:
Changed.